### DISCUSSION TITLE
How to zoom out, without mouse/scrollwheel?

### DISCUSSION THREAD

**Discussion by JanOschii** [webpack-contrib/webpack-bundle-analyzer#563] [Ideas]
Maybe a good info for the readme too.

**Comment by jmm** [comment]
One way I've found to zoom all the way out, at least in Chrome Mac, is to press <kbd>Esc</kbd>. But it'd be really nice to be able to incrementally zoom in / out using the keyboard, and it'd be nice if you could drag a box with the mouse to zoom in. A way to filter by typing path regexes would also be nice.
